src/natcap/invest/sdr/sdr_core.pyx

```diff
@@ -391,11 +419,19 @@ def calculate_sediment_deposition(
     cdef _ManagedRaster sediment_deposition_raster = _ManagedRaster(
         target_sediment_deposition_path, 1, True)
 
+    # given the pixel neighbor numbering system
+    #  3 2 1
+    #  4 x 0
+    #  5 6 7
+    # for a pixel's neighbor in position `i`,
+    # that neighbor's neighbor in position `inflow_offsets[i]`
+    # is the original pixel
```

Comment:
Hmmm, I'm just not quite able to understand this. Does this mean that `inflow_offsets[i]` is `x`?

Reply:
Yeah, I rephrased this a little to hopefully clarify